Makefile

```diff
@@ -28,15 +28,15 @@ stop:  ## Stop the dev server
 
 
 .PHONY: clean
-clean:
+clean: ## Clean up the dev server
 	$(MAKE) stop
 	@docker compose rm --force
 	@docker rmi archivista-archivista --force
 
 
 .PHONY: test
-test:  ## Clean up the dev server
+test: ## Run tests
 	@bash ./test/test.sh
 
 help:  ## Show this help
-	@grep -h -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
\ No newline at end of file
+	@grep -h -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
Mostly curiosity but, I can not figure out the delta between these two lines... what is the change here? 🔎

Reply:
Empty line at the end